documentation/dsi-integration.md

```diff
@@ -0,0 +1,171 @@
+# Authentication
+
+Hiring staff authenticate via the [DfE Sign-in](https://services.signin.education.gov.uk/) system. When [onboarded](./onboarding.md) you are associated with more than one organisation.
+
+Different environments talk to different DfE Sign-in environments:
+
+- Local development, Dev(GOV.UK PaaS) -> Test DfE Sign-in
+- Staging(GOV.UK PaaS) -> Pre-production DfE Sign-in
+- Production (GOV.UK PaaS) -> Production DfE Sign-in
+
+Review apps use the magic link sent via email to authenticate.
+
+## Configure DfE Sign-in Manage Service console
+
+You need to request access to DfE Sign-in Manage Service console for that.
```

Comment:
Please detail how to find the form in service now

Reply:
Added Service Now details, and clarified this whole section